download_data.py

```diff
@@ -20,19 +20,31 @@ def main():
     args = parser.parse_args()
 
     # Get path of repository root folder
-    repo_root_dir = Path(__file__)
-    while REPO_ROOT_NAME not in repo_root_dir.name:
-        repo_root_dir = repo_root_dir.parent
+    repo_root_dir = Path("/grphome/grp_retnet/compute/data") # This is hard coded. You can pass in your own path too
+    # while REPO_ROOT_NAME not in repo_root_dir.name:
+    #     repo_root_dir = repo_root_dir.parent
```

Comment:
Why do we want to leave in the commented code? If we don't need it anymore, shouldn't we just get rid of it?

Reply:
I second the point Nathan made. Also, the const REPO_ROOT_NAME is only used on this line, so it may be best to remove it if we are not using this code.

Reply:
That's fine, we would just be removing functionality for running this in a personal repo, if that makes sense. As in, we don't need it for the group repo, but you might still need it for running it personally. Also, I didn't entirely understand the line, so I didn't just want to throw it out.